src/main/java/com/github/jscookie/javacookie/Cookies.java

```diff
@@ -75,6 +78,9 @@ public synchronized String get( String name ) {
 	@Override
 	public <T> T get( String name, TypeReference<T> typeRef ) throws CookieParseException {
 		String value = get( name );
+		if(value == null) {
```

Comment:
This should be `if ( value == null )` to be consistent with the rest of the file
```suggestion
		if ( value == null ) {
```

It might be worth only release a breaking v1 once we fix this NullPointer hell. Perhaps introducing a [NullObjectPattern](https://en.wikipedia.org/wiki/Null_object_pattern)?

Reply:
Sorry, me IDEA has another idea of Java code formatting, I'll fix it. I would go with `Optional`s to fix the NullPointer hell.